packages/terra-date-input/src/DateInput.jsx

```diff
@@ -5,15 +5,24 @@ import classNamesBind from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
 import uuidv4 from 'uuid/v4';
 import { injectIntl } from 'react-intl';
-import Input from 'terra-form-input';
 import * as KeyCode from 'keycode-js';
 
+import AccessibleValue from 'terra-time-input/lib/_AccessibleValue';
+import AccessibleInput from 'terra-time-input/lib/_AccessibleInput';
 import DateInputUtil from './DateInputUtil';
 import styles from './DateInput.module.scss';
 
 const cx = classNamesBind.bind(styles);
 
 const propTypes = {
+  /**
+   * An identifier used by assistive technologies like screen readers to briefly describe this time input to users.
+   * The label is not rendered visually.
+   *
+   * **BEST PRACTICE FOR ACCESSIBILITY**: you _SHOULD_ set this to match whatever visible label you give in your UI.
```

Comment:
Can we add a UX Accessibility Badge for this prop like below :

<img width="1312" alt="Screen Shot 2022-02-28 at 8 49 24 PM" src="https://user-images.githubusercontent.com/12869809/156008921-e03a6e70-ebfd-4b09-b632-99ffff4da99b.png">


```suggestion
   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue)  **BEST PRACTICE FOR ACCESSIBILITY**: you _SHOULD_ set this to match whatever visible label you give in your UI.
```